js-core/homeworks/homework-9/src/main.js

```diff
@@ -0,0 +1,207 @@
+//Task1
+let developer1 = {
+  skills: ['JavaScript', 'linux', 'html', 'OOP', 'Node.js'],
+  requirements: ['Node.js', 'JavaScript', 'OOP'],
+  goodDev: goodDev
+};
+let developer2 = {
+  experience: [
+    { technology: 'java' },
+    { technology: 'c++' },
+    { technology: 'aws' },
+    { technology: 'docker' }
+  ],
+  requirements: ['java', 'json', 'c++', 'JavaScript'],
+  goodDev: goodDev
+};
+
+function goodDev(dev) {
+	let arrRequirements = this.requirements;
+	if(this.skills) {
+		let arrSkills = this.skills;
+		arrRequirements.forEach(function(elem) {
+			let initialElem = elem;
+			let comparison = arrSkills.some(function(elem) {
```

Comment:
please use arrow function.

You can rename 'elem' inside .some function and give it any better name.

for example it can be requirmentToDeveloper.

So at the end of such function you code would looks like

```js
developerSkill == requirmentToDeveloper
```
That code tolds more to readers